### PR TITLE
docs and examples

### PR DESCRIPTION
**Pull request by abhimutant**

### CODE REVIEW

File: examples/foundationCentral/api_keys/main.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// resources/datasources used in this file were introduced in nutanix/nutanix version 1.5.0-beta.2
+terraform {
+    required_providers {
+      nutanix = {
+          source = "nutanix/nutanix"
+          version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline examples/foundationCentral/api_keys/main.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
 >= as we are releasing this in 1.5.0-beta.2 right ?

File: examples/foundationCentral/imaged_clusters/main.tf [hunk 2]
```diff
@@ -0,0 +1,122 @@
+// resources/datasources used in this file were introduced in nutanix/nutanix version 1.5.0-beta.2
+terraform {
+    required_providers {
+      nutanix = {
+          source = "nutanix/nutanix"
+          version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline examples/foundationCentral/imaged_clusters/main.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
>= as we are releasing this in 1.5.0-beta.2 right ?

File: examples/foundationCentral/imaged_nodes/main.tf [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// resources/datasources used in this file were introduced in nutanix/nutanix version 1.5.0-beta.2
+terraform {
+    required_providers {
+      nutanix = {
+          source = "nutanix/nutanix"
+          version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline examples/foundationCentral/imaged_nodes/main.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
>= as we are releasing this in 1.5.0-beta.2 right ?

File: modules/README.md [hunk 4]
```diff
@@ -28,6 +28,17 @@ module "image_nodes" {
 }
 ```
 
+Example to use foubdation central module
```
Comment (bhati-pradeep) [inline modules/README.md]:
```suggestion
Example to use foundation central module
```

File: modules/foundationCentral/aos-based-node-imaging/node-serial-filter/variables.tf [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+variable "cluster_name" {
+  description = "Name of the cluster"
+  type        = string
+}
+
+//[Optional] : Number of nodes in the cluster. 
+variable "cluster_size" {
+  description = "Number of nodes in the cluster"
+  type        = number
+  default     = 0
```
Comment (bhati-pradeep) [inline modules/foundationCentral/aos-based-node-imaging/node-serial-filter/variables.tf]: can we make this null and let terraform skip it ?

File: modules/foundationCentral/aos-based-node-imaging/node-serial-filter/version.tf [hunk 6]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    nutanix = {
+      source  = "nutanix/nutanix"
+      version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline modules/foundationCentral/aos-based-node-imaging/node-serial-filter/version.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
>= as we are releasing this in 1.5.0-beta.2 right ?

File: modules/foundationCentral/examples/image-aos-nodes/main.tf [hunk 7]
```diff
@@ -0,0 +1,64 @@
+tterraform {
+  required_providers {
+    nutanix = {
+      source  = "nutanix/nutanix"
+      version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline modules/foundationCentral/examples/image-aos-nodes/main.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
>= as we are releasing this in 1.5.0-beta.2 right ?

File: modules/foundationCentral/examples/image-manual-nodes/main.tf [hunk 8]
```diff
@@ -0,0 +1,84 @@
+terraform {
+  required_providers {
+    nutanix = {
+      source  = "nutanix/nutanix"
+      version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline modules/foundationCentral/examples/image-manual-nodes/main.tf]:
```suggestion
          version = ">=1.5.0-beta.2"
```
>= as we are releasing this in 1.5.0-beta.2 right ?

File: modules/foundationCentral/manual-node-imaging/node-serial-filter/output.tf [hunk 9]
```diff
@@ -0,0 +1,3 @@
+output "cls" {
```
Comment (bhati-pradeep) [inline modules/foundationCentral/manual-node-imaging/node-serial-filter/output.tf:2]: add a comment about the output

File: modules/foundationCentral/aos-based-node-imaging/node-serial-filter/output.tf [hunk 10]
```diff
@@ -0,0 +1,3 @@
+output "cls" {
```
Comment (bhati-pradeep) [inline modules/foundationCentral/aos-based-node-imaging/node-serial-filter/output.tf:2]: add a comment about the output

File: modules/foundationCentral/manual-node-imaging/node-serial-filter/version.tf [hunk 11]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    nutanix = {
+      source  = "nutanix/nutanix"
+      version = ">1.5.0-beta.2"
```
Comment (bhati-pradeep) [inline modules/foundationCentral/manual-node-imaging/node-serial-filter/version.tf]:
```suggestion
      version = ">=1.5.0-beta.2"
```

**Review by bhati-pradeep — CHANGES_REQUESTED** [review]

File: examples/foundationCentral/imaged_clusters/main.tf (hunk 2 above)
Comment (abhimutant) [reply examples/foundationCentral/imaged_clusters/main.tf]: yes.

**Review by bhati-pradeep — APPROVED** [review]

**Pull request merged**